src/main/java/org/commcare/suite/model/Detail.java

```diff
@@ -145,6 +145,15 @@ public Detail(String id, DisplayUnit title, String nodeset,
         } else {
             numEntitiesToDisplayPerRow = 1;
         }
+
+        if (relevancy != null) {
+            try {
+                this.parsedRelevancyExpression = XPathParseTool.parseXPath(relevancy);
+            } catch (XPathSyntaxException e) {
+                e.printStackTrace();
```

Comment:
no this being a print statement rather than logging. (also this does error hard, if you want it to error silently i think the below needs to update)